hbw/config/ml_variables.py

```diff
@@ -6,7 +6,6 @@
 
 import order as od
 
-# from columnflow.columnar_util import EMPTY_FLOAT
```

Comment:
I think you did not intend to include these changes since all your dl_ml_inputs should be included in you `add_dl_ml_variables`. Can you revert the changes in this file?